client/tracing/src/lib.rs

```diff
@@ -598,6 +598,7 @@ mod tests {
 				// emit event
 				tracing::event!(target: "test_target", tracing::Level::INFO, "test_event1");
 				for msg in rx.recv() {
+					#[allow(clippy::bool_comparison)]
 					if msg == false {
```

Comment:
```suggestion
					if !msg {
```

Reply:
alas rustc can't quite infer it's a `bool` with `!msg` - I can make the type explicit a little further up.